ci/docker/integration/arrowflight/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Stop background processes when this script exits.
+trap "trap - SIGTERM && kill -- -$$" SIGINT SIGTERM EXIT
+
+python flight_server.py --port 5005 --no-auth &
+python flight_server.py --port 5006 --user=test_user --password=test_password &
```

Comment:
Could you please use a similar password that we have in `tests/integration/helpers/config_cluster.py`? In that way we can detect password leaking to logs. I think the proper password should be `ClickHouse_ArrowFlight_P@ssw0rd`.

Reply:
Please also make sure the password is not logged in:
1. text logs
2. query logs

I will also check once the CI is green and we are close to merging.

Reply:
done